pkg/artifactory/repositories.go

```diff
@@ -1275,25 +1272,26 @@ func composePacker(packers ...PackFunc) PackFunc {
 	}
 }
 
-var defaultPacker = universalPack(noClass)
-
-func inSchema(skeema map[string]*schema.Schema) func(payload interface{}, d *schema.ResourceData) error {
-	return universalPack(schemaHasKey(skeema))
-}
+var defaultPacker = universalPack(nil, noClass)
```

Comment:
I was thinking something more like:

```go
func defaultPacker(skeema map[string]*schema) {
   return univeralPack(hasKey(skeema,noPassword))
}
```

And then not have this code in the universalPack - I think it's unwise to cook that in
}